microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -376,3 +376,12 @@ For the `TopoLVM image`, see link:https://catalog.redhat.com/software/containers
 ==== Bug fixes
 
 * Previously, when {microshift-short} was installed on a non-ostree system, an error was logged during greenboot health check stating that the ostree directory did not exist. {microshift-short} still started normally. Now, the system properly logs the ostree status as “not an ostree system” and an error message is not produced.
+
+[id="microshift-4-14-6-dp"]
+=== RHBA-2023:7684 - {microshift-short} 4.14.6 bug fix update advisory
+
+Issued: 2023-12-12
+
+{product-title} release 4.14.6, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:7684[RHBA-2023:7684] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:7682[RHSA-2023:7682] advisory.
```

Comment:
The two links in this paragraph didn't work for me. I assume that's expected behavior at the moment, but I thought I'd mention it just in case.

Reply:
Yes, they will not work until the errata are pushed live on Wednesday. TY!